rollup.config.js

```diff
@@ -31,7 +32,7 @@ export default {
 				browser: true,
 				dedupe: ['svelte']
 			}),
-			commonjs(),
+			commonjs(!dev && { sourceMap: false }),
```

Comment:
What kind of performance gains are you generally seeing from this? Is that build speed improvement worth the reduced prod debugging fidelity? I guess this is rather project specific so I'm not sure what the default should be.

Additionally, this could be simplified:

```suggestion
			commonjs({ sourceMap: dev }),
```